### PR TITLE
Linear models

### PR DESCRIPTION
**Pull request by MikeLippincott**
Look deeper into the visualization of linear models of single cell morphology features.

### CODE REVIEW

File: 3.sc_Morphology_Linear_Modeling/README.md
Comment (axiomcura) [inline 3.sc_Morphology_Linear_Modeling/README.md:26]:
```suggestion
I have included a visualization from [CFReT Analysis](https://github.com/gwaybio/CFReT_data/blob/368f0baf7b027f5614de7173bbc7336add81fcb3/4.analyze-data/notebooks/linear_model/1.visualize_linear_model.ipynb) to visluaize the beta coefficients across morphology feature types and cellular compartments:
```

File: 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r
Comment (axiomcura) [inline 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r:81]: Consider adding some documentation here.

File: 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r
Comment (axiomcura) [inline 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r:134]: same comment above

**Review by axiomcura — APPROVED** [review]
LGTM, added some suggestions. 

I've noticed that the R scripts are primarily meant for plotting; however, it is difficult to follow what is being plotted. I do not know if adding some documentation about the plot is necessary. Furthermore, I'll leave that up to you! Otherwise, great work!

File: 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r
Comment (MikeLippincott) [reply 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r:81]: I might remove this in the future, I am holding off on extra changes for now

File: 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r
Comment (MikeLippincott) [reply 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r:134]: same here, not a strong analysis

**Pull request merged**
